webtool/test_transcribe.py

```
@@ -827,3 +827,36 @@ def flush(self): pass
+
+def test_transcribe_project_meldet_active_done_und_ueberspringt_geloeschtes_audio(monkeypatch, tmp_path, capsys):
+    proj_dir = tmp_path / "Demo"
+    audio_dir = proj_dir / "audio"
+    audio_dir.mkdir(parents=True)
+    f1 = audio_dir / "S1.mp3"
+    f1.write_bytes(b"audio1")
+    f2 = audio_dir / "S2.mp3"
+    f2.write_bytes(b"audio2")
+
+    monkeypatch.setattr(transcribe, "PROJEKTE", str(tmp_path))
+    monkeypatch.setattr(transcribe, "_modell", lambda *a, **kw: "fake_model")
```

Ruff (0.16.1)

[warning] Unused lambda argument: `a`

(ARG005)